### PR TITLE
Responsiveness For Profile and Blob Page

### PR DESCRIPTION
**Pull request by harkiratsm**
@Abhishek-kumar09 

**fixed Responsiveness of Profile and blob home page**

**Bugs Fixed ✅**

1. Responsiveness
2. profile Section Description 
3. Avatar Profile section
4. and other minor bugs

### CODE REVIEW

**Review by Abhishek-kumar09 — COMMENTED** [review]
Run `npm run format` to make the code better.

**Comment by harkiratsm** [comment]
@Abhishek-kumar09 Should Close this pr and again pull a draft pr

File: src/components/Profile.js [hunk 1]
```diff
@@ -41,21 +41,30 @@ export default function Profile() {
...
+        <Grid
+          item
+          xs={12}
+          sm={5}
+          style={{ marginRight: '21px', marginBottom: '21px' }}
+        >
           <CommitChart />
         </Grid>
-        <Grid item sm>
+        <Grid item xs={12} sm>
```
Comment (Abhishek-kumar09) [inline src/components/Profile.js]: sm has no values set?
if you want to set some values for sm,  do it. Otherwise, remove the sm prop

**Comment by harkiratsm** [comment]
@Abhishek-kumar09  Done

**Pull request merged**